UPGRADE-5.1.md

```diff
@@ -7,6 +7,13 @@ FrameworkBundle
  * Marked `MicroKernelTrait::configureRoutes()` as `@internal` and `@final`.
  * Deprecated not overriding `MicroKernelTrait::configureRouting()`.
 
+HttpFoundation
+--------------
+
+ * Deprecate `Response::create()`, `JsonResponse::create()`,
```

Comment:
should be `Deprecated` for consistency (all messages are using the past  in the upgrade guide)

Reply:
I'm slowly moving to use more "modern" message patterns as use everywhere else.